src/utils/currencies.tsx

```diff
@@ -77,3 +77,6 @@ export const generateWalletSourceInformation = (
 // till Sepolia v0.0.10: 21118
 export const PREVIOUS_TOTAL_USERS = 21118;
 export const COIN_GECKO_SOURCE = 'https://www.coingecko.com/en/coins/';
+
+export const handlePriceSource = (asset: CurrencySymbol | undefined) =>
+    asset && COIN_GECKO_SOURCE.concat(currencyMap[asset].coinGeckoId);
```

Comment:
Why do we need to use CoinGecko again?
As far as I remember, we have replaced all prices of CoinGecko with prices of Chainlink before because our smart contracts refer to Chainlink prices.

Reply:
Yeah this is an old/existing method. I moved it to currencies.tsx. Should we be using something like this instead: https://data.chain.link/feeds/ethereum/mainnet/btc-usd ? @biga816

Reply:
There should be already logic in out front-end code tot get the Chainlink price through our smart contract.

Reply:
`getLastPrice()` is the function for that.

Reply:
This functionality leads the user to the place where we get our prices.
In the past it was CoinGecko.

Reply:
I suppose this was a way for users to look at a price source. If this is an issue, this should be covered outside of this ticket.

Reply:
If this update is to show the price sources of our protocol, it's a wrong update.

Reply:
It isn't to show the price sources. This method existed [here already](https://github.com/Secured-Finance/secured-finance-app/blob/5e1cdd073f239a7f3c6f931fa298b9f0c683b99b/src/components/molecules/AdvancedLendingTopBar/AdvancedLendingTopBar.tsx#L206) and all I did was to move it to utils. If we want to change the price source, it isn't covered in this ticket.

Reply:
Understand.
Could you create a ticket to improve price source links?

Reply:
Got it. I've created it [here](https://secured-finance.atlassian.net/jira/software/projects/SF/boards/1?assignee=unassigned&selectedIssue=SF-1262)